### ISSUE TITLE
Can't finish installation 

### ISSUE THREAD

**Issue by cesarhgd10** [despark/ignicms#55]
after running` php artisan igni:admin:install
`
I get this error, I did everything right by the documentation 

_Generating autoload files_
                                                                  
 _[Symfony\Component\Console\Exception\CommandNotFoundException]  
 There are no commands defined in the "igni:admin" namespace._

**Comment by rupeshkarki** [comment]
I also found got the following error upon running command php artisan igni:admin:install
[Symfony\Component\Debug\Exception\FatalErrorException]
  Class 'Despark\Cms\Providers\FieldServiceProvider' not found
so upon checking Providers folder, there doesn't seem to exist FieldServiceProvider.php file.

**Comment by cesarhgd10** [comment]
Yes, I was getting this error too. Seems like the packagist repository is different from Github's, so I copied the Despark\Cms\Providers\FieldServiceProvider from here and pasted in my application, then I started to get the error above.

**Comment by astoimenov** [comment]
Hi guys,

I'm really sorry but there was an error in the readme of the master branch. The **FieldServiceProvider** is included only in the master branch itself and we don't have it in any release yet. If you want to try the CMS I'll recommend you to change the version of the package to _dev-master_, because there we fixed many things that were in the previous releases. Although the master branch is in active development and it'll be updated frequently.

Thank you for the issue opened and that you gave our **ignicms** a try.
